kernel/include/stdbool.h

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
I've added this file to handle the basic definition of `boolean` types, if we have an independent OS, it has to have all headers and declarations using a non-standard libc.